platform/osgi-web-util/pom.xml

```diff
@@ -56,6 +56,10 @@
             <artifactId>org.motechproject.org.codehaus.jackson</artifactId>
         </dependency>
         <dependency>
+            <groupId>org.springframework.security</groupId>
```

Comment:
Do we need this dependency? Why not have the code that determines the tab in server-bundle? Seems to belong more there